mlir/lib/Dialect/Affine/Transforms/AffineParallelize.cpp

```diff
@@ -0,0 +1,69 @@
+//===- Affineparallelize.cpp - Affineparallelize Pass
+//-------------------------===//
+//
+// Copyright 2019 The MLIR Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// =============================================================================
+//
+// This file implements a new parallelizer for affine loop nests that is able to
```

Comment:
Please don't use "new". The existing one was just a parallelism *detection* test not a parallelization.